src/mpl/configure.ac

```diff
@@ -842,15 +834,9 @@ case $with_proc_mutex_package in
 	    if test "$pac_cv_cpu_set_defined" = "yes" ; then
 	        AC_DEFINE(HAVE_CPU_SET_MACROS,1,[Define if CPU_SET and CPU_ZERO defined])
             fi
-        # FIXME: Some versions of sched_setaffinity return ENOSYS (!),
-	    # so we should test for the unfriendly and useless behavior
         fi
     fi
 
-    if test "$ac_cv_func_pthread_setaffinity_np" = "yes" ; then
-      AC_DEFINE(HAVE_PTHREAD_SETAFFINITY_NP,1, [Define if pthread_setaffinity_np is available.])
```

Comment:
Is it because the AC_CHECK_FUNCS already defines this macro?

Reply:
Yeah. This configure.ac stuff could probably split another commit. I looked at for a while before realizing so much of it was useless or redundant. Deleted it reflexively 😅.